src/Symfony/Component/Form/Extension/Validator/Constraints/FormValidator.php

```diff
@@ -25,6 +25,7 @@
 class FormValidator extends ConstraintValidator
 {
     private $resolvedGroups;
+    private $fieldFormConstraints;
```

Comment:
We keep all the created `Form` constraint instances until the whole form tree has been validated to prevent PHP from reusing object hashes/id as we use them internally in the Validator component to create cache keys (see also #36415).

Reply:
Does this replace #36415?

Reply:
Nope, this is just a workaround that can be reverted once we have solved #36415 (I am going to look into it now).

Reply:
see #38387